util/action.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$id": "https://kulturkrock.se/action.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Action in Opus",
+    "description": "An action object in an Opus to use for Screencrash",
```

Comment:
I don't this description says much. How about something like "An action such as starting or stopping an effect"? Same in the other places like this.